worlds/noita/Locations.py

```diff
@@ -0,0 +1,229 @@
+# Locations are specific points that you would obtain an item at.
+from enum import IntEnum
+from typing import Dict, NamedTuple, Optional
+from BaseClasses import Location
+
+
+class NoitaLocation(Location):
+    game: str = "Noita"
+
+
+class LocationData(NamedTuple):
+    id: int
+    ltype: Optional[str] = ""
+    flag: int = 0
+
+
+class Orbs(IntEnum):
+    main_path = 1
+    side_path = 2
+    main_world = 3
+    parallel_worlds = 4
+
+
+class Bosses(IntEnum):
+    main_path = 1
+    side_path = 2
+    main_world = 3
+    parallel_worlds = 4
+
+
+class HiddenChests(IntEnum):
+    main_path = 1
+    side_path = 2
+    main_world = 3
+    parallel_worlds = 4
+
+
+class Pedestals(IntEnum):
+    main_path = 1
+    side_path = 2
+    main_world = 3
+    parallel_worlds = 4
+
+
+# Mapping of items in each region
+# Only the first Hidden Chest and Pedestal are mapped here, the others are created in Regions
+# ltype key: "hc" = Hidden Chests, "peds" = Pedestals, "boss" = Boss, "orb" = Orb.
```

Comment:
We can make the following changes to improve readability for outsiders:

`"hc"` -> `"chest"`
`"peds"` -> `"pedestal"`